### PR TITLE
Newsweek Clone for Microverse

### PR DESCRIPTION
**Pull request by warblo001**
This is a project for the Microverse program.

In this project, I created a clone of the Newsweek.com site using the next front-end technologies :

The project has the following features:

- Responsiveness
- Bootstrap
- Flexbox and Floats
- Media queries for different screen sizes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1284 @@
...
+                    class="abigail"
+                  />
+                  <h1>
+                    The Transgender Threat to Women's Sports
+                    <br />
+                    <span class="by">BY </span
+                    ><span class="abtext">ABIGAIL SHRIER</span>
+                  </h1>
+                  <div class="vs"
+                    ><img src="assets/img/vs.jpg" alt="" />
```
Comment (umairarshadbutt) [inline index.html]: Please add the text in `alt` attribute

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1284 @@
...
+          </h1>
+          <p>Fri, Oct 23, 2020</p>
+        </div>
+
+        <div class="logo">
+          <img src="assets/img/logo.jpg" alt="logo" />
+        </div>
+
+        <div class="login">
+          <img src="assets/img/login.jpg" alt="login" />
```
Comment (umairarshadbutt) [inline index.html]: Please remove the image and design by HTML and CSS

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1284 @@
...
+        </div>
+      </div>
+
+      <nav>
+        <div class="nav-logo">
+          <img src="assets/img/smallogo.jpg" alt="logo" />
+        </div>
+        <div class="nav-login1">
+          <img src="assets/img/login2.jpg" alt="login" />
```
Comment (umairarshadbutt) [inline index.html]: Please remove the image and design by HTML and CSS

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1284 @@
...
+        </div>
+      </nav>
+
+      <div class="first-nav">
+        <div class="nav-logo">
+          <img src="assets/img/smallogo.jpg" alt="logo" />
+        </div>
+        <div class="nav-login1">
+          <img src="assets/img/login3.jpg" alt="login" />
```
Comment (umairarshadbutt) [inline index.html]: Please remove the image and design by HTML and CSS

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1284 @@
...
+              class="iconify"
+              data-icon="mdi:weather-partly-cloudy"
+              data-inline="false"
+            ></span>
+            20° Terrassa, ES >
+          </h1>
+          <p>Fri, Oct 23, 2020</p>
+        </div>
+
+        <div class="logo">
```
Comment (umairarshadbutt) [inline index.html:42]: Please add the Navigation Link for Logo Image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1284 @@
...
+                >
+                  <h1>FEATURED STORIES</h1>
+
+                  <div class="featured-stories">
+                    <img
+                      src="assets/img/hunter-biden-tony-bobulinski-wall-street-journal.webp"
+                      alt="biden"
+                    />
+                    <h2>U.S.</h2>
+                    <h3>
```
Comment (umairarshadbutt) [inline index.html]: Please add the Navigation Link. (Please add the Navigation Link where we are using the pointer. Update the navigation link on the complete page)

**Review by umairarshadbutt — COMMENTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @warblo001
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes  👇.
- [x] Navigation Links are missing in the navbar and main section. Please add the navigation links.
- [x] Please remove the images of **LOGIN**, **SUBSCRIBE** and **Hamburger**. And design it by using HTML, CSS and Bootstrap.
- [x] Please add the text in `alt` attribute. (see the inline comment)
- [x] Screenshot is missing in README file. please add the screenshot of the landing page.

- [x] **[OPTIONSL]:** You can use [Multi Devices Mockup](https://techsini.com/multi-mockup/index.php) for the screenshot.


##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

File: assets/style.css [hunk 7]
```diff
@@ -0,0 +1,1604 @@
...
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.news-text button {
+  background: #f72210;
+  width: 80%;
+  height: 38px;
```
Comment (descholar-ceo) [inline assets/style.css]: You are repeating yourself here, you have alredy use this `heigth: 38px;` on line 287, 1087, 1123 :point_right: create a utility class which have this property and apply it wherever you need this property

File: assets/style.css [hunk 8]
```diff
@@ -0,0 +1,1604 @@
...
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.news-text button {
+  background: #f72210;
```
Comment (descholar-ceo) [inline assets/style.css]: You are repeating yourself here, you have alredy use this `background: #f72210;` on line 44, 122, 152, 161, 652, 873, 1085, 1121 point_right create a utility class which have this property and apply it wherever you need this property

File: index.html [hunk 9]
```diff
@@ -0,0 +1,1451 @@
...
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+      integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
+      crossorigin="anonymous"
+    />
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+      integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous"></script>
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
```
Comment (descholar-ceo) [inline index.html]: It's not a good idea to load your script in head tag, it is nice to put this script loading at the end of you html codes inside body before body closing tag

File: index.html [hunk 10]
```diff
@@ -0,0 +1,1451 @@
...
+          <a class="nav2-a" href="#">Sports</a>
+          <a class="nav2-a" href="#">Health</a>
+          <a class="nav2-a" href="#">The Debate</a>
+          <a class="nav2-a" href="#">Vantage</a>
+          <a class="nav2-a" href="#">Weather</a>
+        
+        </div>
+        <input type="search" placeholder="Search" />
+        <span class="iconify" data-icon="ei:search" data-inline="false"></span>
+        <img src="assets/img/pesta.jpg" alt="s" />
```
Comment (descholar-ceo) [inline index.html]: The alt attribute should have a short descriptive text as its value

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @warblo001  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [x] Since this project is mainly focusing on bootstrap, using display flex, or grid, in your own CSS/SCSS files is highly discouraged, try to replace them by bootstrap utility classes, [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png), you can refer to the following links to learn how to use [grid with bootstrap](https://getbootstrap.com/docs/4.3/layout/grid/) and [flexbox](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options)
- [x] In your CSS file, you have some properties that you have written again and again, please don't repeat yourself, you should either used utility classes and give them the propert you wish to apply, and then apply that class on any element you wish, or you can even group elements which are sharing the same property, [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png)
- [x] Implement all changes I suggested in inline comments in your codes

# [Optional changes]
- [x] It's nice that you implemented the feedback you got of adding `alt` on all img tags, but make sure that all `alt` have descriptive text as their values e.g.: `alt=programmer's picture`
- [x] Your commit messages should be in imperative mode or present tense, you should have used `Remove extra files, fix linters issues` instead of `Removed extra files, fixed linters issues`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://adoring-benz-430942.netlify.app/) shows a good understanding and usage of Bootstrap. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @warblo001! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**